lib/google/longrunning/operations_client.rb

```diff
@@ -347,6 +318,24 @@ def delete_operation \
         @delete_operation.call(req, options)
         nil
       end
+
+      protected
+
+      def default_settings(metadata, lib_name, lib_version)
+        package_version = Gem.loaded_specs['google-gax'].version.version
+
+        google_api_client = ["gl-ruby/#{RUBY_VERSION}"]
+        google_api_client << "#{lib_name}/#{lib_version}" if lib_name
+        google_api_client << "gapic/#{package_version}"
+        google_api_client << "gax/#{Google::Gax::VERSION}"
+        google_api_client << "grpc/#{GRPC::VERSION}"
+        google_api_client.join " "
+
+        headers = { 'x-goog-api-client' => google_api_client }
```

Comment:
No longer need to freeze the `google_api_client` string? Will frozen strings be default from the supported Ruby versions?

Reply:
Correct. A future change will add google-style and the magic frozen string comment.